src/styles/clock.scss

```diff
@@ -0,0 +1,54 @@
+.hand {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  width: 4px;
+  height: 55px;
+  background-color: #000;
+  transform-origin: top center;
+}
+
+.stopwatch {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  width: 200px;
+  height: 200px;
+  border: 10px inset #000;
+  border-radius: 50%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  &__minute {
+    transform: rotate(0deg) translateX(-50%) translateY(-100%);
+    animation: rotate-minute 3600s linear infinite;
+  }
+
+  &__second {
+    background-color: red;
+    width: 2px;
+    height: 80px;
+    transform: rotate(0deg) translateX(-50%) translateY(-100%);
+    animation: rotate-second 60s linear infinite;
+  }
+}
+
+@keyframes rotate-minute {
+  from {
+    transform: rotate(0deg) translateX(-50%) translateY(-100%);
+  }
+  to {
+    transform: rotate(360deg) translateX(-50%) translateY(-100%);
+  }
+}
+
+@keyframes rotate-second {
+  from {
+    transform: rotate(0deg) translateX(-50%) translateY(-100%);
+  }
+  to {
+    transform: rotate(360deg) translateX(-50%) translateY(-100%);
+  }
+}
```

Comment:
move to another file
and why do we need two the same keyframes

Reply:
> move to another file and why do we need two the same keyframes

i used two for minutes and seconds